$NetBSD: patch-at,v 1.8 2010/05/20 08:06:44 sbd Exp $

Don't create the empty driver dir, do that with OWN_DIRS

--- scheduler/Makefile.orig	2007-02-05 23:20:52.000000000 +1300
+++ scheduler/Makefile
@@ -106,7 +106,9 @@ depend:
 # Install the scheduler...
 #
 
-install:	all
+install:	installbin
+
+installbin:	all
 	echo Creating $(SBINDIR)...
 	$(INSTALL_DIR) -m 755 $(SBINDIR)
 	echo Installing cupsd in $(SBINDIR)...
@@ -120,6 +122,8 @@ install:	all
 	$(INSTALL_BIN) cups-driverd $(SERVERBIN)/daemon
 	$(INSTALL_BIN) cups-lpd $(SERVERBIN)/daemon
 	$(INSTALL_BIN) cups-polld $(SERVERBIN)/daemon
+
+installdir:
 	echo Creating $(SERVERBIN)/driver...
 	$(INSTALL_DIR) -m 755 $(SERVERBIN)/driver
 	echo Creating $(SERVERROOT)...
